lattice_bscan_spi.py

```diff
@@ -125,7 +126,6 @@ def elaborate(self, platform):
             ]
 
         m.d.comb += [
-            cd_sync.rst.eq(self.jtag_sel1_capture),
```

Comment:
This line was intended to adopt the same clock domain reset logic as the Xilinx counterpart ([line 87](https://github.com/quartiq/bscan_spi_bitstreams/blob/fe3a91a48fd207da2dd681dfc6332727bea54eb6/xilinx_bscan_spi.py#L87)). Has there been any negative effect brought by this line?

Reply:
Update: another user has recently adapted the proxy bitstream for their MachXO2 FPGA, and they also reported that the removal of this sync clock reset is necessary to deassert CSn. @ilya-epifanov please keep your change here :smiley: